File: 프로그래머스/lv1/86051. 없는 숫자 더하기/없는 숫자 더하기.java
class Solution {
    public int solution(int[] numbers) {
        int missingSum = 0;

        for (int i = 0; i <= 9; i++) {
            boolean found = false;
            for (int num : numbers) {
                if (num == i) {
                    found = true;
                    break;
                }
            }
            if (!found) {
                missingSum += i;
            }
        }

        return missingSum;
    }
}

File: 프로그래머스/lv1/86051. 없는 숫자 더하기/없는 숫자 더하기.js
function solution(numbers) {
    let missingSum = 0;

    for (let i = 0; i <= 9; i++) {
        let found = false;
        for (let j = 0; j < numbers.length; j++) {
            if (numbers[j] === i) {
                found = true;
                break;
            }
        }
        if (!found) {
            missingSum += i;
        }
    }

    return missingSum;
}